### PR TITLE
Merge latest changes to main

### PR DESCRIPTION
**Pull request by YuelinXin**
### Main changes:
1. Add CSS variables;
2. Email testing function updates

### CODE REVIEW

File: application/static/CSS/main.css [hunk 1]
```diff
@@ -5,7 +5,32 @@
...
+    --mf-border-black: rgb(50, 50, 50);
+    --mf-button-yes: rgb(83, 162, 88);
+    --mf-button-yes-border: rgb(82, 153, 87);
+    --mf-button-yes-hover: rgb(78, 149, 83);
+    --mf-button-no: rgb(186, 71, 62);
+    --mf-button-no-border: rgb(220, 96, 85);
+    --mf-button-no-hover: rgb(180, 63, 56);
+    --mf-header-purple-gradient: linear-gradient(120deg, rgb(194, 172, 233), rgb(76, 9, 234));
+    --link-blue: rgb(42, 100, 196);
+}
```
Comment (YuelinXin) [inline application/static/CSS/main.css:32]: root variables are added to improve maintainability

**Review by YuelinXin — COMMENTED** [review]
All changes have been self-reviewed by the author himself

**Review by Bilgin-L — APPROVED** [review]
Approved

**Review by Bilgin-L — COMMENTED** [review]
Approved

**Pull request merged**